configuration.go

```diff
@@ -47,6 +47,7 @@ type Configuration struct {
 	ClientAuthKeyString  string
 	CAString             string
 	Insecure             bool
+	SkipSessionAuth      bool
```

Comment:
What the default value for this if we don't specify this value? Also, we previously are using token-based auth right? What is the suggested auth when we connect with NSX?

Reply:
The default is false, so no change in behavior if this value is not specified